libs/module-to-standalone/user/shell/src/lib/user-shell.component.ts

```diff
@@ -27,6 +28,8 @@ import { Component, Inject } from '@angular/core';
   host: {
     class: 'flex flex-col p-4 gap-3 border border-blue',
   },
+  standalone: true,
+  imports: [RouterLink, RouterOutlet],
 })
 export class UserShellComponent {
   constructor(@Inject(TOKEN) public token: string) {}
```

Comment:
you should use the new inject function, especially for replacing @Inject(), this way you have the typing.